CHANGES.rst

```diff
@@ -4,6 +4,10 @@ Changes
 3.2 (unreleased)
```

Comment:
```suggestion
4.0 (unreleased)
```

Reply:
I'd bump the release number because of the Python 3.7 drop.

Reply:
At some point we said that dropping support for a sunset Python version does not require a new major version. That's why I implemented this in `meta/config/update-python-version.py`.

Reply:
OK, I must have forgotten that discussion. I still bump the version whenever I touch packahes. It doesn't really matter, though. It's just a number.